pkg/catalog/prepare.go

```diff
@@ -0,0 +1,132 @@
+package catalog
```

Comment:
does this belong in the catalog package? props worth its own package

Reply:
Yeah, thought about that. It might warrant its own package. But I thought every command just sort of goes there. :D I can refactor. :)